### PR TITLE
Streamlining and modifying docs in the commons repository

### PR DESCRIPTION
**Pull request by Preeticp**
This PR:
. Modifies the main readme to focus on testing related information.
. Modifies titles of the other docs in the various folders to relevant titles rather than just readmes.
. Converts markdown to asciidoc and makes relevant format corrections.
. Two of the docs need content, I will work on them when I get the content for the same.

### CODE REVIEW

**Comment by tisnik** [comment]
@Preeticp could you please try to resolve the merge conflict?

**Comment by tisnik** [comment]
[test]

**Comment by Preeticp** [comment]
@tisnik I have resolved the conflicts, please check.

File: integration-tests/integration_tests.adoc [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+This repository contains integration tests for the fabric8-analytics services.
+
+The tests can be run against existing deployment, or locally by using docker-compose.
+
+The following environment variables can be used to test specific deployments:
+
+* `F8A_API_URL` - API server URL
+* `F8A_JOB_API_URL` - Jobs service URL
+
+By default, the system running on the localhost is tested. The <tests?> are not containerized currently hence they start and stop Fabric8 Analytics multiple times. Thus the following prerequisites need to be met:
```
Comment (tisnik) [inline integration-tests/integration_tests.adoc]: yes, the tests, well specifically integration tests

**Comment by Preeticp** [comment]
@tisnik I have made the required change. I do not have the rights to merge, request you to merge the PR.
Remaining changes in another PR sounds good. Thank you and welcome :)

**Comment by tisnik** [comment]
Ah sure @Preeticp. Do you want to be added to the fabric8 group?
Anyway - I'm merging this PR, thank you very much.

**Comment by Preeticp** [comment]
Sure @tisnik please add me to the group.

**Pull request merged**